### ISSUE TITLE
Private repos don't work

### ISSUE THREAD

**Issue by marcinwyszynski** [DeckHub/App#22]
Even though the app was granted permissions for private repos no activity from these is reported in the organization view and private repos can't be added.

- Deckhub Version: 1.0.0 
- Platform/OS & Version: OS X 10.11.3

**Comment by burakcan** [comment]
Hi @marcinwyszynski, in which format you're searching repo? You should only search with 'reponame', not 'org/reponame'.
But we're aware of this problem if your repo name is too generic, it's almost impossible to find by searching from there. We are going to add an option that allows user to manually provide a repo url or 'org/reponame'

**Comment by marcinwyszynski** [comment]
Not really. I have an organization called thecodebeat with one public repo
and 4 private ones. I can't see activity from any of the private repos.

On Tuesday 19 April 2016, Burak Can <notifications@github.com> wrote:

> Hi @marcinwyszynski <https://github.com/marcinwyszynski>, in which format
> you're searching repo? You should only search with 'reponame', not
> 'org/reponame'.
> But we're aware of this problem if your repo name is too generic, it's
> almost impossible to find by searching from there. We are going to add an
> option that allows user to manually provide a repo url or 'org/reponame'
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DeckHub/App/issues/22#issuecomment-211755616>
>


-- 
Marcin

**Comment by burakcan** [comment]
Can you please try adding a "Home" column for the organization to see if this is a authentication problem or something else?

**Comment by marcinwyszynski** [comment]
Exactly what I did:

![deckhub](https://cloud.githubusercontent.com/assets/571786/14633902/0238ea90-0620-11e6-9b21-0827b2e7a199.png)

**Comment by burakcan** [comment]
Thank you @marcinwyszynski,
I marked this issue for our upcoming release. Expect an update in the following days. Sorry for inconvenience. 

Happy developing :)

**Comment by jordanjay29** [comment]
Please update, searching all repos without the org name is useless and does not bring up my private repo.
